importer/api/__init__.py

```diff
@@ -0,0 +1,100 @@
+"""
+Import OLX data into LORE.
+"""
+
+# stdlib
+from glob import glob
+from shutil import rmtree
+from tempfile import mkdtemp
+from os.path import join
+import tarfile
+import zipfile
+
+# PyPi
+
+from xbundle import XBundle, DESCRIPTOR_TAGS
+from lxml import etree
+
+# MITODL
+# commented out for now, until it's merged
+# from lore.learningobjects.models import Course, LearningObject
+
+def import_course_from_file(filename):
```

Comment:
This is because we will be getting a filename when the course is uploaded as a tarball. If we're getting it from elsewhere we'll still have to save it as a file before opening it, because that's how `zipfile` and `tarfile` work.